src/main/java/com/github/dockerjava/core/DockerClientConfig.java

```diff
@@ -418,8 +418,8 @@ public final DockerClientConfigBuilder withDockerConfig(String dockerConfig) {
         }
 
         public final DockerClientConfigBuilder withDockerTlsVerify(String dockerTlsVerify) {
-            this.dockerTlsVerify = BooleanUtils.toBoolean(dockerTlsVerify.trim())
-                    || BooleanUtils.toBoolean(dockerTlsVerify.trim(), "1", "0");
+            String trimmed = dockerTlsVerify.trim();
+            this.dockerTlsVerify = "true".equals(trimmed) || "1".equals(trimmed);
```

Comment:
Maybe `org.apache.commons.lang.BooleanUtils#toBooleanObject(java.lang.String)` is too complicated in this case, `"true".equals(trimmed) || "1".equals(trimmed)` is simple, fast and straight forward.